project/.travis/install_lint.sh

```diff
@@ -9,3 +9,4 @@ chmod u+x "${HOME}/bin/php-cs-fixer"
 composer global require sllh/composer-lint:@stable --prefer-dist --no-interaction
 
 gem install yaml-lint
+sudo apt-get install -y libxml2-utils
```

Comment:
Are you sure it's working? Sudo is not activated on Travis AFAIK.

Reply:
Let me check on adminbundle

Reply:
It does not work indeed. Do I need to enable sudo?

Reply:
I would prefer not. It decrease performance a lot.

No another way to get this binary?

Reply:
I am trying other way to install it